quickjs.c

```diff
@@ -40487,12 +40520,14 @@ static JSValue js_iterator_helper_next(JSContext *ctx, JSValue this_val,
     JSIteratorHelperData *it;
     JSValue ret;
 
+    *pdone = FALSE;
```

Comment:
This should fix the unitialized jump valgrind caught: https://github.com/quickjs-ng/quickjs/actions/runs/11786411039/job/32829640312